src/confetti.js

```diff
@@ -286,8 +288,8 @@
     return {
       x: opts.x,
       y: opts.y,
-      wobble: Math.random() * 10,
-      wobbleSpeed: Math.min(0.11, Math.random() * 0.1 + 0.05),
+      wobble: opts.wobble,
+      wobbleSpeed: typeof opts.wobbleSpeed === 'function' ? opts.wobbleSpeed() : opts.wobbleSpeed,
```

Comment:
Can you tell me more about why this is a function?

Reply:
From my understanding the random value is supposed to be evaluated whenever its requested. If i "hardcode" the random value, the confetti will wobble the same way. I wasnt sure if this was the correct way of doing this. Do you have a better way of doing this? :)

Reply:
After further inspection i agree that having this as a function is weird. Instead ive made this a value and then times that with `Math.random`